doc/rst/developer/bestPractices/ContributorInfo.rst

```diff
@@ -195,19 +285,548 @@ Mail chapel-developers@ for review and merge
 
 * For the day or two after the commit has gone in, check the
   chapel-test-results-regressions_ mailing list to ensure that there are no new
-  failures (or that they are due to a commit other than yours).  Use the
-  chapel-developers_ mailing list if you are unsure (a member of the core
-  Chapel team will be tasked with diagnosing any testing failures on any given
-  night, but it's nice when developers notice the issue first themselves to
-  save wasted effort).
+  failures caused by your commit.  Use the chapel-developers_ mailing list if
+  you are unsure (a member of the core Chapel team will be tasked with
+  diagnosing any testing failures on any given night, but it's nice when
+  developers notice the issue first themselves to save wasted effort).
+
+.. _HOWTO/Git/GitHub details:
+
+HOWTO/Git/GitHub details
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. _initial git setup:
+
+Initial Git Setup
++++++++++++++++++
+
+Follow the GitHub directions to setup a new account.
+
+https://help.github.com/categories/53/articles
+
+If you plan to use ssh to push/pull, setup SSH keys.
+
+https://help.github.com/categories/56/articles
+
+
+
+.. _Configure your local git:
+
+Configure your local git
+++++++++++++++++++++++++
+
+Here is the uncommented version of the commands:
```

Comment:
I don't understand what "the uncommented version of the commands" is supposed to mean. There are comments in there.